pkg/objectkinds/registry.go

```diff
@@ -18,6 +18,16 @@ func registerObjectKind(name string, objectKind Matcher) {
 	allObjectKinds[name] = objectKind
 }
 
+// AllObjectKinds will return all the object kind names that are registered
+func AllObjectKinds() []string {
+	var kinds []string
```

Comment:
```suggestion
	kinds := make([]string, 0, len(allObjectKinds))
```

Reply:
I somehow figured you would want that.  I had it before but removed.  I'll add back

Reply:
Ah, nm.  You made the change.